iidm/iidm-api/src/main/java/com/powsybl/iidm/network/ZipLoadModelAdder.java

```diff
@@ -0,0 +1,30 @@
+/**
+ * Copyright (c) 2023, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ * SPDX-License-Identifier: MPL-2.0
+ */
+package com.powsybl.iidm.network;
+
+/**
+ * @author Geoffroy Jamgotchian <geoffroy.jamgotchian at rte-france.com>
+ */
+public interface ZipLoadModelAdder {
+
+    double SUM_EPSILON = 1E-8; // tolerance on coefficient sum
```

Comment:
Why do you need this coefficient?

Reply:
because when you sum doubles it is sometimes not possible to reach exactly 1